packages/juce-blueprint/src/components/Canvas.ts

```diff
@@ -0,0 +1,116 @@
+import React, { Component } from 'react';
```

Comment:
Arguably we should have some file/class description comment up at the top here. Not sure what pattern you would prefer moving forward for the individual component files?

Reply:
Yes! I don't have any pattern in mind, but let's start rolling those in on a future PR